pkg/scalers/aws_sqs_queue_scaler.go

```diff
@@ -115,6 +115,8 @@ func parseAwsSqsQueueMetadata(config *ScalerConfig, logger logr.Logger) (*awsSqs
 
 	if val, ok := config.TriggerMetadata["queueURL"]; ok && val != "" {
 		meta.queueURL = val
+	} else if val, ok := config.TriggerMetadata["queueURLFromEnv"]; ok && val != "" {
+		meta.queueURL = config.ResolvedEnv[val]
```

Comment:
You need to check that `config.ResolvedEnv[val]` is not blank, so that we're not assigning a blank queueURL. If it's blank, we should report the same error as no url provided.

Reply:
I've updated it with a check on that